docs/guide.md

```diff
@@ -334,7 +334,21 @@ packages. SciCookie support the following:
   build requirements. With its capabilities, it facilitates cross-platform
   builds using CMake and effortless integration with C/C++ libraries, making
   it a valuable asset for research software engineers. 
-  
+
+- [**Pybind11**](https://pybind11.readthedocs.io/en/stable/): It's build system designed
```

Comment:
Should we call it something like `setuptools + pybind11` to be more clear, given that `pybind11` itself is not a backend?

Reply:
Or atleast using "setuptools + pybind11" in all the user-facing locations, and just "pybind11" in the internals of `scicookie`.

Reply:
> Or atleast using "setuptools + pybind11" in all the user-facing locations, and just "pybind11" in the internals of `scicookie`.

that makes sense to me.

Reply:
Wouldn't that be confusing since we are using setuptools as default build-sytem?

Reply:
not really ... setuptools is the default option because it was the first one historically, but a lot of people want to kill that. That is why a bunch of project are looking for new alternatives.
but for c/c++ code setuptools is still a great option. 
so specifying setuptools + pybind11 makes a lot of sense, because it is clear about its structure.